src/cmds/qsub.c

```diff
@@ -5640,6 +5633,10 @@ main(int argc, char **argv, char **envp) /* qsub */
 	char **argv_cpy; /* copy argv for getopt */
 	int i;
 
+	if (initsocketlib()) {
```

Comment:
Why are we moving this from set_sig_handlers() ?

Reply:
1. Initializing socket library operation is not related to signal handling.
2. It is not possible to make it platform independent there

Reply:
Yes agree, I was thinking why it was like that. But as you noticed in other part of the code also we are initializing in main, so to make it consistent, it will be better to move in main.